nextcloudappstore/core/models.py

```diff
@@ -47,8 +47,11 @@ class Meta:
     def __str__(self):
         return self.name
 
-    def allows_editing(self, user):
```

Comment:
Django permissons uses can_*. allows_* suggest that the user allows someone to edit. It doesn't sound like a check if edting is allowed.